workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/public-sites-navigation/index.js

```diff
@@ -5,6 +5,10 @@
 
 /* eslint-disable no-undef */
 
+const groupPathFriendlyURLPublic = document.querySelector(
```

Comment:
Instead of doing this we should call methods from the Liferay object. Something like Liferay.ThemeDisplay.getLayoutURL()

Reply:
I'll try using "a href="/" before trying this